Libplanet.Tests/Net/Protocols/TestTransport.cs

```diff
@@ -80,6 +80,8 @@ internal class TestTransport : ITransport
 
         public IEnumerable<BoundPeer> Peers => Protocol.Peers;
 
+        public DateTimeOffset LastMessageTimestamp { get; private set; }
```

Comment:
```suggestion
        /// <summary>
        /// The last time when any message was arrived.  It can be <see cref="DateTimeOffset"/>'s
        /// epoch (i.e., <c>default(DateTimeOffset)</c>) if no message has been arrived yet.
        /// </summary>
        public DateTimeOffset LastMessageTimestamp { get; private set; }
```